src/render/extract.rs

```diff
@@ -96,10 +96,8 @@ impl ExtractedTilemapTexture {
                     it is being extracted as a texture!",
                 );
                 let texture_size: TilemapTextureSize = image.size().into();
-                let tile_count_x =
-                    (texture_size.x + tile_spacing.x) / (tile_size.x + tile_spacing.x);
-                let tile_count_y =
-                    (texture_size.y + tile_spacing.y) / (tile_size.y + tile_spacing.y);
+                let tile_count_x = (texture_size.x) / (tile_size.x + tile_spacing.x);
```

Comment:
Is this making the assumption that every tile, including the last one in the atlas will have padding? Is that the right assumption here?

Reply:
That is indeed the assumption.

If we wanted to get rid of this assumption, I propose that we take inspiration from bevy's own bevy_sprite `from_grid_with_padding`.

Reply:
@antoinePaulinB7 I have been thinking about it, and I think it's a fine assumption, because it provides a lot of regularity and eliminates special cases. Perhaps it is something we can document, so users know that this is what we expect. I will create a separate issue for it if this PR gets merged, to work on documenting that.